scripts/build.sh

```diff
@@ -2,7 +2,7 @@
 
 set -euo pipefail
 
-GOOS="linux" go build -ldflags='-s -w' -tags osusergo -o bin/main github.com/paketo-buildpacks/gradle/v6/cmd/main
+GOARCH="amd64" GOOS="linux" go build -ldflags='-s -w' -tags osusergo -o bin/main github.com/paketo-buildpacks/gradle/v6/cmd/main
```

Comment:
Was this intentional? We're going to need to figure out how to handle architecture soon, but I'd rather not start adding bits in until we've got a plan so we can apply the same across all of the buildpacks. Unless there's a specific need for this now. Thanks

Reply:
no sorry, it was not.

just that I use that on my machine, and forgot to remove it committing